modules/openapi-generator/src/test/resources/3_0/issue_19393_map_of_inner_enum.yaml

```diff
@@ -0,0 +1,51 @@
+openapi: 3.0.0
+info:
+  version: 1.0.0
+  title: OpenAPI Test API
+  license:
+    name: Apache-2.0
+    url: 'https://www.apache.org/licenses/LICENSE-2.0.html'
+paths:
+  /employees/{employeeId}:
+    put:
+      operationId: updateEmployee
+      parameters:
+        - name: employeeId
+          in: path
+          required: true
+          schema:
+            type: string
+            format: uuid
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Employee'
+        required: true
+      responses:
+        '200':
+          description: OK
+components:
+  schemas:
+    Employee:
+      type: object
+      properties:
+        projectRole:
+          type: object
+          additionalProperties:
+            type: string
+            enum:
+              - DEVELOPER
+              - TESTER
+              - OWNER
+        projectRoles:
+          type: object
+          additionalProperties:
+            uniqueItems: true
+            type: array
+            items:
+              type: string
+              enum:
+                - DEVELOPER
+                - TESTER
+                - OWNER
```

Comment:
i did a test with this spec but still got errors:
```
[ERROR] C:\Users\wing3\AppData\Local\Temp\java-enum\src\main\java\org\openapitools\client\model\Employee.java:[118,9] error: enum InnerEnum is already defined in class Employee
[ERROR] C:\Users\wing3\AppData\Local\Temp\java-enum\src\main\java\org\openapitools\client\JSON.java:[49,7] error: duplicate class: org.openapitools.client.JSON

```

can you please take a look?

command: 
```
java -jar modules/openapi-generator-cli/target/openapi-generator-cli.jar generate -g java -i modules/openapi-generator/src/test/resources/3_0/issue_19393_map_of_inner_enum.yaml -o /tmp/java-enum/
```

Reply:
Oh, sorry about that! 

The errors seem to be due to a _different_ bug where the same name (`InnerEnum`) gets generated for multiple enums within the same schema. 

I've now updated the test spec to use two different schemas.